Userland/Utilities/install.cpp

```diff
@@ -14,11 +14,13 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
 {
     TRY(Core::System::pledge("stdio rpath wpath cpath fattr"));
 
+    bool copy_mode = true;
     bool create_leading_dest_components = false;
     StringView source;
     StringView destination;
 
     Core::ArgsParser args_parser;
+    args_parser.add_option(copy_mode, "Copy files (default)", nullptr, 'c');
```

Comment:
Would `ArgsParser::add_ignored` work here?

Reply:
Indeed that works. I have added a comment with an explanation anyways.